app/views/devise/invitations/new.html.erb

```diff
@@ -1,40 +1,120 @@
-    <%= f.select :organisation_id, organisation_options(f), organisation_select_options, { class: "chosen-select form-control", 'data-module' => 'chosen' } %>
-  </p>
-
-  <h2 class="add-vertical-margins">Permissions</h2>
-  <%= render partial: "shared/user_permissions", locals: { user_object: f.object }%>
-
-  <%= f.submit "Create user and send email", :class => 'btn btn-success' %>
-<% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for resource, as: resource_name, url: invitation_path(resource_name), html: { method: :post } do |f| %>
+      <% if f.object.errors.any? %>
+        <%= render "govuk_publishing_components/components/error_summary", {
+          id: "error-summary",
+          title: "There was a problem with your new user",
+          items: f.object.errors.map do |error|
+            {
+              text: error.full_message,
+              href: "#user_#{error.attribute}",
+            }
+          end
+        } %>
+      <% end %>
+
+      <%= render "govuk_publishing_components/components/input", {
+        label: { text: "Name" },
+        name: "user[name]",
+        id: "user_name",
+        error_items: f.object.errors.full_messages_for(:name).map { |message| { text: message } },
+        value: f.object.name,
+        autocomplete: "off",
+      } %>
+
+      <%= render "govuk_publishing_components/components/input", {
+        label: { text: "Email" },
+        name: "user[email]",
+        id: "user_email",
+        error_items: f.object.errors.full_messages_for(:email).map { |message| { text: message } },
+        value: f.object.email,
+        autocomplete: "off",
+      } %>
+
+      <%= render "govuk_publishing_components/components/select", {
+        id: "user_organisation_id",
+        name: "user[organisation_id]",
+        label: "Organisation",
+        options: options_for_organisation_select(selected: f.object.organisation_id)
+      } %>
+
+      <% if policy(User).assign_role? %>
+        <% role_hint = capture do %>
+          <%= render "govuk_publishing_components/components/list", {
+            visible_counters: true,
+            items: [
+              "<strong>Superadmins</strong> can create and edit all user types and edit applications.",
+              "<strong>Admins</strong> can create and edit normal users.",
+              "<strong>Super Organisation Admins</strong> can unlock and unsuspend their organisation and related organisation accounts.",
+              "<strong>Organisation Admins</strong> can unlock and unsuspend their organisation accounts.",
+            ]
+          } %>
+        <% end %>
+
+        <%= render "govuk_publishing_components/components/select", {
+          id: "user_role",
+          name: "user[role]",
+          label: "Role",
+          hint: role_hint,
+          options: options_for_role_select(selected: f.object.role),
+        } %>
+      <% end %>
+
```

Comment:
In the [figma designs](https://www.figma.com/file/ArJZK2wjwRwIHJItpRTGjq/GOV.UK-Publishing-SignOn---June-2023?type=design&node-id=865-1658&mode=design) for this page, when we eventually split it into two, there's a "Manage Permissions" title for the second page with the permission select boxes. I wonder if it's worth adding a similar header here to break up the page and make it more scannable as an interim step.

Reply:
> In the [figma designs](https://www.figma.com/file/ArJZK2wjwRwIHJItpRTGjq/GOV.UK-Publishing-SignOn---June-2023?type=design&node-id=865-1658&mode=design) for this page, when we eventually split it into two

My current understanding is that we're not _definitely_ going to split this page, but I may have got the wrong end of the stick. Joseph was less concerned about splitting *this* page versus the batch invitation page, because the latter has even more going on with the CSV, etc.

> there's a "Manage Permissions" title for the second page with the permission select boxes. I wonder if it's worth adding a similar header here to break up the page and make it more scannable as an interim step.

Good point. Looking at the diff, I've lost an "Permissions" `h2` heading, so I'll add something like that back before merging.

Reply:
Addressed in https://github.com/alphagov/signon/pull/2412/commits/45c8da72c26ae363ae73f8495455d61b637eff57